lib/domainPolice/index.js

```diff
@@ -25,7 +25,7 @@ function domainMatches (domain, host) {
     return index >= 0 && index === (1 + host.length - domain.length);
   }
 
-  return host === domain || host === ("www." + domain);
+  return host === domain || host === ('www.' + domain);
```

Comment:
I was fixing a linting issue with double-quotes. Semicolons don't have a project-wide rule. This module uses semicolons. So no, I will not make that decision for this module.